src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -500,7 +501,7 @@ describe('AwsSamDebugConfigurationProvider', async () => {
                     uri: vscode.Uri.file(appDir),
                 },
                 baseBuildDir: actual.baseBuildDir, // Random, sanity-checked by assertEqualLaunchConfigs().
-                codeRoot: pathutil.normalize(path.join(appDir, 'src/HelloWorld')), // Normalized to absolute path.
+                codeRoot: expectedCodeRoot, // Normalized to absolute path.
```

Comment:
Since [this decision](https://github.com/aws/aws-toolkit-vscode/pull/1078#discussion_r421186238) was moved to the CONFIG phase it is now clearly visible in the non-integ tests.